Cargo.toml

```diff
@@ -274,8 +274,12 @@ debug = true
 # to analyze performance with tooling like linux perf
 [profile.ci]
 inherits = "dev"
-strip = false
-debug = false
+strip = "none"
+debug = 0
+debug-assertion = false
+lto = "off"
+panic = "abort"
```

Comment:
Due to our `catch_unwind` handling for login shell stability there is a restraining order against any uses of `panic = "abort"`